src/Toolkit/Toolkit.Maui/BookmarksView/BookmarksView.cs

```diff
@@ -192,16 +167,15 @@ private void SelectAndNavigateToBookmark(Bookmark bookmark)
     /// <summary>
     /// Handles selection on the underlying list view.
     /// </summary>
-    private void Internal_bookmarkSelected(object? sender, SelectedItemChangedEventArgs e)
+    private void Internal_bookmarkSelected(object? sender, SelectionChangedEventArgs e)
     {
-        if (e.SelectedItem is Bookmark bm)
-        {
-            SelectAndNavigateToBookmark(bm);
-        }
-
-        if (e.SelectedItem != null && sender is ListView lv)
+        if (sender is CollectionView cv)
         {
-            lv.SelectedItem = null;
+            if (cv.SelectedItem is Bookmark item)
+            {
+                SelectAndNavigateToBookmark(item);
+            }
+            cv.ClearValue(CollectionView.SelectedItemProperty);
```

Comment:
I don't think you need to do this. Instead in the template set `SelectionMode='None'`. I believe you can still click it.

Reply:
By default, Selection mode of `CollectionView` is `None` which actually disables the `SelectionChanged` event.
So we can not get to hit this handler.